app/views/ansible_variables/index.html.erb

```diff
@@ -26,7 +26,7 @@
           variable.ansible_role.name,
           hash_for_ansible_variables_path(:search => "ansible_role = #{variable.ansible_role}")
         ) %></td>
-        <td class="ellipsis"><%= "String" %></td>
+        <td class="ellipsis"><%= variable.key_type || _('string') %></td>
```

Comment:
should `string` be really translated? if key_type was string, it wouldn't be translated, correct?

Reply:
Correct, I removed the translation.